src/main/java/com/google/alpollo/Song.java

```diff
@@ -2,35 +2,26 @@
 
 import com.googlecode.objectify.annotation.Index;
 import com.googlecode.objectify.annotation.Entity;
-import com.googlecode.objectify.annotation.Id;
 
 /** 
  * Represents the structure of the song. 
  * Please do not add fields because they will be saved in the search counter database. 
  */
 @Entity
 public class Song {
-  @Id private String id;
```

Comment:
Same question: why not to store this id and get rid of song counter class?

Reply:
Because I don't want to save counter as a part of song